cosmic/utils.py

```diff
@@ -514,7 +514,7 @@ def newton_cotes(x, f) :
 
 
 def rndm(a, b, g, size):
-    """Power-law generator for pdf(x)\propto x^{g} for a<=x<=b
+    """Power-law generator for pdf(x)\propto x^{g-1} for a<=x<=b
```

Comment:
again, I think things would be much clear throughout the code if  rndm takes in the powerlaw exponent directly